editor/src/messages/portfolio/document/document_message_handler.rs

```diff
@@ -1435,6 +1435,20 @@ impl MessageHandler<DocumentMessage, DocumentMessageContext<'_>> for DocumentMes
 						},
 					})
 				}
+				// Some parts of the editior depend on these bounds to be present
```

Comment:
what parts? Ideally the editor should not be aware of the bounds at all, since it treats the top left corner of the viewport at (0,0). Only the executor should know the bounds in order to evaluate the node graph.